### PR TITLE
DOC-1811 - RC: Name field updates for account & profile

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
Staged previews are available:
- [Manage user profile & settings](https://docs.redis.com/staging/jira-doc-1811/rc/accounts/user-profile/)
- [Access management](https://docs.redis.com/staging/jira-doc-1811/rc/security/access-management/#add-user), specifically the _Add user_ & _Edit user_ sections

File: content/rc/accounts/user-profile.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ To review your user profile settings, select **User profile** from the Profile c
 
 This screen contains up to three sections, including:
 
-- The *User details* section includes basic information about your account, including _Name_, _Job title_, _Email_, and the date the account was created.  _Name_ and _Job title_ can be edited; the other settings are read-only.
+- The *User details* section includes basic information about your account, including _First Name_, _Last Name_, _Job title_, _Email_, and the date the account was created.  The name and job title can be edited; other settings are read-only.
```
Comment (rrelledge) [inline content/rc/accounts/user-profile.md:36]:
```suggestion
- The *User details* section includes basic information about your account, including _First name_, _Last name_, _Job title_, _Email_, and the date the account was created.  The names and job title can be edited; other settings are read-only.
```

File: content/rc/security/access-management.md [hunk 2]
```diff
@@ -51,13 +51,14 @@ If you have a large team, you can use the controls in the list footer to navigat
...
-{{<image filename="images/rc/access-management-add-user-dialog.png" width="50%" alt="Use the Add User dialog to specify the details for your new user." >}}{{< /image >}}
+{{<image filename="images/rc/access-mgmt-add-user-dialog.png" width="50%" alt="Use the Add User dialog to specify the details for your new user." >}}{{< /image >}}
 
 Use this dialog to specify the following values:
 
 | Setting | Description |
 |---------|-------------|
-| **Name** | Name of the user displayed in the admin console and in email messages |
+| **First Name** | First name of the user displayed in the admin console and in email messages |
+| **Last Name** | Last name of the user displayed in the admin console and in email messages |
```
Comment (rrelledge) [inline content/rc/security/access-management.md:61]:
```suggestion
| **First name** | First name of the user displayed in the admin console and in email messages |
| **Last name** | Last name of the user displayed in the admin console and in email messages |
```

**Review by rrelledge — APPROVED** [review]
Looks good to me.

**Comment by lanceleonard** [comment]
#hold-off until GA

File: content/rc/security/access-management.md (hunk 2 above)
Comment (lanceleonard) [reply content/rc/security/access-management.md:61]: Fixed.

File: content/rc/accounts/user-profile.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/accounts/user-profile.md:36]: Fixed.

**Pull request merged**